docs/pages/Deploying/Hosting-the-app.md

```diff
@@ -6,47 +6,40 @@ Here's a step-by-step guide on how to setup an Amazon Lightsail instance to host
 
 (If you know how to create a Lightsail instance, you can skip to the recommended configuration part by clicking here).
 
-### 1. Create an account or login to https://lightsail.aws.amazon.com
+### 1. Create an AWS Account: 
+If you haven't already, create or log in to your AWS account at lightsail.aws.amazon.com.
 
-### 2. Click on "Create instance"
+### 2. Create an Instance: 
 
-### 3. Create your instance
+a. Click "Create Instance."
 
-The first step is to select the "Instance location". In most cases, there's no need to switch locations as the default one will work well.
+b. Select the "Instance location." In most cases, the default location works fine.
 
-After that, it is time to pick your Instance Image. We recommend using "Linux/Unix" as the image and "Ubuntu 20.04 LTS" as the Operating System.
+c. Choose "Linux/Unix" as the image and "Ubuntu 20.04 LTS" as the Operating System.
 
-As for instance plan, it'll vary depending on your unique demands, but a "1 GB, 1vCPU, 40GB SSD and 2TB transfer" setup should cover most scenarios.
+d. Configure the instance plan based on your requirements. A "1 GB, 1vCPU, 40GB SSD, and 2TB transfer" setup is recommended for most scenarios.
 
-Lastly, identify your instance by giving it a unique name and then hit "Create instance".
+e. Give your instance a unique name and click "Create Instance."
 
-PS: Once you create your instance, it'll likely take a few minutes for the setup to be completed.
+PS: It may take a few minutes for the instance setup to complete.
 
-#### The recommended configuration is as follows:
+### Connecting to Your newly created Instance
 
-- Ubuntu 20.04 LTS
-- 1GB RAM
-- 1vCPU
-- 40GB SSD Hard Drive
-- 2TB transfer
```

Comment:
You need to make sure you specify that its ubuntu somewhere

Reply:
Hi, i've mentioned this in point c

![image](https://github.com/arc53/DocsGPT/assets/104289350/d859d8d7-7b12-40b7-b4e8-84a1ead4be67)

should i specify it again?